main.cpp

```diff
@@ -1792,6 +1796,8 @@ static int main_inner(int argc, char *argv[])
 	});
 	cbs.add("--msl-runtime-array-rich-descriptor",
 	        [&args](CLIParser &) { args.msl_runtime_array_rich_descriptor = true; });
+	cbs.add("--msl-replace-recursive-inputs",
+			[&args](CLIParser &) { args.msl_replace_recursive_inputs = true; });
```

Comment:
Indentation is off.

Reply:
Not in my IDE. Must just be a tab-spacing issue. I've copied the spacing from the line above.